### PR TITLE
Chunk Screen + Tasks on TL

### PR DESCRIPTION
**Pull request by tlelievre26**
## What is this PR introducing?

Basically introduces everything left on our frontend, including a "zoomed in" screen for chunks, an app-specific metrics screen, and tasks on the timeline. This probably should have been broken up into multiple PRs but whatever

## What components of our system does this change impact? Which core feature is this relevant to?

Frontend

## Does this change depend on any other unmerged PRs? Link them here if so.

No

## Pre-commit to-do list

- [x] Does this PR only contain one feature that cannot be broken up into smaller components?
- [ ] Have you written unit/integration tests for this feature?
- [x] Have you run the linter over your code for this change?
- [ ] Have you run your changes against the testing suite?
- [x] Have you given a written explanation of the change?
- [x] Have you asked a teammate for review?
- [x] **Did you update the version number appropriately**

### CODE REVIEW

File: src/timeline/components/metrics-components/FullSessionMetrics.jsx [hunk 1]
```diff
@@ -25,22 +29,22 @@ const FullSessionMetrics = () => {
             <p># Tasks Completed: {sessionMetrics.numTasks}</p>
           </div>
           <div>
-            <p>Tab Switches per min: {sessionMetrics.tabSwitchRate}</p>
+            <p>Tab Switches per min: {sessionMetrics.tabSwitchRate.toFixed(2)}</p>
```
Comment (stephzhan) [inline src/timeline/components/metrics-components/FullSessionMetrics.jsx:32]: what's toFixed for?

**Review by stephzhan — APPROVED** [review]
nice

File: src/timeline/components/metrics-components/AppUsageTimeBlock.jsx
[the remark this answers is not in the document]
Comment (tlelievre26) [reply src/timeline/components/metrics-components/AppUsageTimeBlock.jsx:1]: Oh this was just me deleting a file

File: src/timeline/components/metrics-components/FullSessionMetrics.jsx (hunk 1 above)
Comment (tlelievre26) [reply src/timeline/components/metrics-components/FullSessionMetrics.jsx:32]: Basically just truncates the decimal (so 2.333333 -> 2.33)

**Pull request merged**